Student_BCOs/(FinalUpdate)Juntian_Wei_BCO_000549.json

```diff
@@ -0,0 +1,434 @@
+                    }
+                ],
+                "prerequisite": [
+                    {
+                        "name": "R 3.0.1 or higher",
+                        "uri": {
+                            "access_time": "",
+                            "uri": "http://mirror.fcaglp.unlp.edu.ar/CRAN/",
+                            "sha1_checksum": "000528checksum",
+                            "filename": ""
+                        }
+                    }
+                ],
+                "description": "This step is for differential expression analysis"
+            },
+            {
+                "name": "gplots",
+                "version": "3.0.1.1",
+                "step_number": 3,
+                "input_list": [],
+                "output_list": [],
+                "prerequisite": [],
+                "description": "gplots is used to generate plot for differential analysis and cluster analysis"
+            },
+            {
+                "name": "limma",
+                "version": "3.14.4",
+                "step_number": 3,
+                "input_list": [],
+                "output_list": [],
+                "prerequisite": [],
+                "description": "Limma is used to perform quantile normalization for differential expression analysis. "
+            },
+            {
+                "name": "No dependencies",
+                "version": "",
+                "step_number": 4,
+                "input_list": [],
+                "output_list": [],
+                "prerequisite": [],
+                "description": "The final step has no dependencies. TargetScan and miRanda are used, which are two different websites. TargetScan: http://www.targetscan.org/vert_72/. miRanda: http://www.microrna.org/microrna/home.do"
+            }
+        ]
+    },
+    "execution_domain": {
+        "external_data_endpoints": [
+            {
+                "name": "MiRanda",
+                "url": "http://www.microrna.org/microrna/home.do"
+            },
+            {
+                "name": "miRanalyzer subsection 3.1",
+                "url": "https://bioinfo2.ugr.es/miRanalyzer/standalone.html#toc-Subsection-3.1"
+            }
+        ],
+        "environment_variables": {},
+        "script_driver": "",
+        "software_prerequisites": [
+            {
+                "name": "Python",
+                "version": "2.7 or higher",
+                "uri": {
+                    "access_time": "",
+                    "uri": "https://www.python.org/getit/",
+                    "sha1_checksum": "000528checksum",
+                    "filename": ""
+                }
+            },
+            {
+                "name": "PERL",
+                "version": "5.0 or higher",
+                "uri": {
+                    "access_time": "",
+                    "uri": "http://www.perl.org/get.html",
+                    "sha1_checksum": "000528checksum",
```

Comment:
The ID for the submitted BCO is BCO_000549. 
This should be "000549_checksum". See #34 
Please modify this checksum accordingly.